### PR TITLE
Add block-info tp in rust

### PR DESCRIPTION
**Pull request by eloaverona**
Adds block info tp in rust and makes is compatible to wasm -- by @ltseeley 

Signed-off-by: Eloá Franca Verona <eloafran@bitwise.io>

### CODE REVIEW

**Comment by rbuysse** [comment]
bin/build_debs and bin/build_rust will be removed by #1729. The tp should be added to docker-compose-installed.yaml and docker/compose/copy-debs.yaml instead.

**Comment by peterschwarz** [comment]
Why is this being closed?

**Comment by eloaverona** [comment]
I closed this one and started a new [PR](https://github.com/hyperledger/sawtooth-core/pull/1767).  I was experience some git difficulties, and figure this would be easier way around it.

File: .gitignore [hunk 1]
```diff
@@ -50,7 +50,13 @@
 /families/settings/sawtooth_settings/protobuf/
 /families/smallbank/smallbank_go/src/protobuf/
 /families/smallbank/smallbank_go/bin/
-/families/block_info/sawtooth_block_info/protobuf
+/families/block_info/python/sawtooth_block_info/protobuf
+/families/block_info/python/.DS_Store
```
Comment (gilescope) [inline .gitignore:54]: you don't want .DS_Store - thats just mac cruft.

**Pull request closed without merging**